modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-order-test/src/testIntegration/java/com/liferay/headless/commerce/admin/order/resource/v1_0/test/OrderRuleAccountGroupResourceTest.java

```diff
@@ -58,53 +56,40 @@ public void setUp() throws Exception {
 			testCompany.getCompanyId(), testGroup.getGroupId(),
 			_user.getUserId());
 
-		DateConfig displayDateConfig = DateConfig.toDisplayDateConfig(
-			RandomTestUtil.nextDate(), _user.getTimeZone());
-		DateConfig expirationDateConfig = DateConfig.toExpirationDateConfig(
-			RandomTestUtil.nextDate(), _user.getTimeZone());
-
 		_corEntry = _corEntryLocalService.addCOREntry(
 			RandomTestUtil.randomString(), _user.getUserId(),
-			RandomTestUtil.randomBoolean(), RandomTestUtil.randomString(),
-			displayDateConfig.getMonth(), displayDateConfig.getDay(),
-			displayDateConfig.getYear(), displayDateConfig.getHour(),
-			displayDateConfig.getMinute(), expirationDateConfig.getMonth(),
-			expirationDateConfig.getDay(), expirationDateConfig.getYear(),
-			expirationDateConfig.getHour(), expirationDateConfig.getMinute(),
-			true, RandomTestUtil.randomString(), 0,
+			RandomTestUtil.randomBoolean(), RandomTestUtil.randomString(), 1, 1,
+			2022, 12, 0, 0, 0, 0, 0, 0, true, RandomTestUtil.randomString(), 0,
 			RandomTestUtil.randomString(), StringPool.BLANK, _serviceContext);
 	}
 
+	@Ignore
 	@Override
 	@Test
 	public void testDeleteOrderRuleAccountGroup() throws Exception {
+		super.testDeleteOrderRuleAccountGroup();
 	}
 
+	@Ignore
 	@Override
 	@Test
 	public void testGraphQLDeleteOrderRuleAccountGroup() throws Exception {
-	}
-
-	@Override
-	protected Collection<EntityField> getEntityFields() throws Exception {
-		return new ArrayList<>();
+		super.testGraphQLDeleteOrderRuleAccountGroup();
 	}
 
 	@Override
 	protected OrderRuleAccountGroup randomOrderRuleAccountGroup()
 		throws Exception {
 
-		CommerceAccountGroup commerceAccountGroup =
-			_commerceAccountGroupLocalService.addCommerceAccountGroup(
-				_user.getCompanyId(), RandomTestUtil.randomString(), 0, false,
-				null, _serviceContext);
+		AccountGroup accountGroup1 = _accountGroupLocalService.addAccountGroup(
```

Comment:
Why the name is accountGroup1?

Reply:
I changed the way we create the OrderRuleAccountGroup